### DISCUSSION TITLE
Welcome to Namaste-BlockChain Discussions!

### DISCUSSION THREAD

**Discussion by CODEX108** [CODEX108/Namaste-BlockChain#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our project. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cdx108** [comment]
Get Set GO!
